pkgs/development/python-modules/radio_beam/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, astropy }:
+
+buildPythonPackage rec {
+  pname = "radio_beam";
+  version = "0.2";
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
The tests require several `pytest` plugins that are not in `nixpkgs`. I've added a comment in the expression.

Reply:
The tests require several `pytest` plugins that are not in `nixpkgs`. I've added a comment in the expressions.